client/components/shopping-cart/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import NumberInput from 'components/number-field/number-input';
+import Gridicon from 'components/gridicon';
+import { sanitize } from 'dompurify';
+
+const ShoppingCart = ( { packages, updateValue, dimensionUnit, weightUnit, errors } ) => {
```

Comment:
Instead of `ShoppingCart`, which sounds like a front side component, could we use something like `OrderContents` for the name for this component instead?